graphql-koa-dataloader/app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "apollo-koa-dataloader",
+  "version": "1.0.0",
+  "description": "Example application using Apollo, Koa, and a Dataloader",
+  "main": "server.js",
+  "dependencies": {
+    "@apollo/server": "^4.7.1",
+    "@as-integrations/koa": "^1.0.0",
+    "@faker-js/faker": "^8.0.0",
+    "@koa/cors": "^4.0.0",
+    "dataloader": "^2.2.2",
+    "graphql": "^16.6.0",
+    "graphql-tag": "^2.12.6",
+    "koa": "^2.14.2",
+    "koa-bodyparser": "^4.4.0",
+    "mongodb": "^5.5.0",
```

Comment:
we don't support mongo v5 just yet. I would downgrade and re-test. you might be starting off with an incorrect baseline

Reply:
Thanks.

This really should have been a WIP, as there were other problems, such as not using the ESM loader, nor the Apollo plugin. I'm just going to rebase to add those.